theano/compile/mode.py

```diff
@@ -422,6 +393,7 @@ def get_mode(orig_string):
             # This might be required if the string is 'ProfileMode'
             from .profilemode import ProfileMode  # noqa
             from .profilemode import prof_mode_instance_to_print
+            # TODO: Can't we look up the name and invoke it rather than using eval here?
```

Comment:
Probably. There is only the modes ProfileMode (deprecated, we can probably remove it now) and MonitorMode that aren't the default Mode.
Mostly, if we add the elif for MonitorMode and Mode and remove ProfileMode, then we could remove that eval.